packages/cli/doc-plugin-typedoc/src/sidebar.ts

```diff
@@ -0,0 +1,74 @@
+import path from 'path';
+import fs from '@modern-js/utils/fs-extra';
+import {
+  getClassPath,
+  getModulePath,
+  getInterfacePath,
+  getFunctionPath,
+} from './utils';
+
+export async function resolveSidebar(jsonDir: string): Promise<any[]> {
```

Comment:
No `any`

Reply:
Because the type cannot be imported from doc-core due to module tools build error.